sdk/resourcemanager/Azure.ResourceManager/CHANGELOG.md

```diff
@@ -1,14 +1,22 @@
 # Release History
 
-## 1.0.0-beta.9 (Unreleased)
+## 1.0.0-beta.9 (2022-03-31)
 
 ### Features Added
 
-### Breaking Changes
+- New struct `ArmEnvironment`.
 
-### Bugs Fixed
+### Breaking Changes
 
-### Other Changes
+- Now all the resource classes will have a `Resource` suffix (if it previously does not have one).
+- Rename some models to more comprehensive names.
```

Comment:
Maybe `Renamed` is better? Because the previous changelogs all used the past tense.

Reply:
I jsut checked the changelog of azure.core. They are using past tense.
I will change the tense here